app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -359,28 +363,40 @@ public Survey getSurveyMostRecentlyPublishedVersion(StudyIdentifier studyIdentif
         return new QueryBuilder().setStudy(studyIdentifier).isPublished().setSurvey(guid).isNotDeleted().getOne(true);
     }
     
-    // SCAN
-    @Override
-    public Survey getSurveyMostRecentlyPublishedVersionByIdentifier(StudyIdentifier studyIdentifier, String identifier) {
-        return new QueryBuilder().setStudy(studyIdentifier).setIdentifier(identifier).isPublished().isNotDeleted().getOne(true);
-    }
```

Comment:
This can be supported by the studyKey index. So unless we're sure we won't need this at all...